scenes/lessons/7th_class/miandam/miandam.py

```diff
@@ -0,0 +1,578 @@
+                ReplacementTransform(modified_equation[12].copy(), two_plus_four_times_70[3]),
+                Write(two_plus_four_times_70[0]),
+                Write(two_plus_four_times_70[2]),
+                Write(two_plus_four_times_70[4]),
+                Write(two_plus_four_times_70[5]),
+                Write(two_plus_four_times_70[6])
+            )
+            self.wait(0.5)
+
+            self.play(
+                ReplacementTransform(two_plus_four_times_70[:-2], final_equation[4]),
+                ReplacementTransform(two_plus_four_times_70[-2], final_equation[5]),
+                ReplacementTransform(two_plus_four_times_70[-1], final_equation[6]),
+                Write(final_equation[3])
+            )
+            self.wait(0.25)
+            self.play(FadeOut(surr_rects_70))
+            self.wait()
+
+        def anim_adding_variables(): # Ուրիշ 2 հոգի, ուրիշ արագություններ
+            self.play(
+                ReplacementTransform(aram_60[0], urish),
+                aram_60[1:].animate.next_to(urish, buff=0.25),
+                ReplacementTransform(babken_70[0], urish_urish),
+                babken_70[1:].animate.next_to(urish_urish, buff=0.25)
+            )
+            self.wait()
+            
+            self.play(
+                Transform(aram_60[2], seventy.move_to(aram_60[2])),
+                Transform(babken_70[2], ninty.move_to(babken_70[2]))
+            )
+            self.wait()
+
+            self.play(
+                *[Transform(mob, seventy.copy().move_to(mob)) for mob in sixties],
+                *[Transform(mob, ninty.copy().move_to(mob)) for mob in seventies]
+            )
+            self.wait()
+
+            self.play(*[Transform(mob, x.copy().move_to(mob).align_to(mob, DOWN)) for mob in sixties])
+            self.wait()
+            self.play(*[Transform(mob, y.copy().move_to(mob).align_to(mob, DOWN).shift(0.1 * DOWN)) for mob in seventies])
+            self.wait()
+
+            self.play(Circumscribe(final_equation, fade_out=True, run_time=2))
+            self.wait()
+
+            car.clear_updaters()
+            moving_dot.clear_updaters()
+            self.play(FadeOut(
+                aram_60, babken_70, first_day_equation,
+                modified_equation, car, moving_dot, road,
+                segments_a, segments_b, urish, urish_urish
+            ))
+            self.wait(0.5)
+
+            self.remove(final_equation)
+            self.play(
+                Transform(
+                    VGroup(final_equation[0], *final_equation[2:5], final_equation[-1]), 
+                    Tex('5', 'x', ' + ', '6', 'y', font_size=70)
+                ),
+                FadeOut(final_equation[1], final_equation[-2])
+            )
+            self.wait()
+
+        anim_hamarotagrutyun()
+        anim_first_day()
+        anim_modify_equation()
+        anim_second_day()
+        anim_final_equation()
+        anim_adding_variables()
```

Comment:
Why is having those functions better then writing the code without the function?

Reply:
զուտ տեսքը ինձ սենց ավելի հարմար ա, հետո կորդը կարդալուց էլ ինձ թվում ա ավելի հեշտ ա, ահագին լավ նկարագրում են թե որ կտորն ինչ ա անում